docs/gradle-plugin.md

```diff
@@ -0,0 +1,82 @@
+(gradle-plugin-page)=
+
+# Using the Nextflow Gradle plugin
+
+The [Nextflow Gradle plugin](https://github.com/nextflow-io/nextflow-plugin-gradle) simplifies plugin development by configuring default dependencies needed for Nextflow integration and defining Gradle tasks for building, testing, and publishing Nextflow plugins.
+
+:::{note}
+The Nextflow Gradle plugin and plugin registry are currently available as a private beta. See the {ref}`migration guide <migrating-plugin-page>` for more information.
+:::
+
+(gradle-plugin-create)=
+
+## Creating a plugin
+
+:::{versionadded} 25.04.0
+:::
+
+The easiest way to get started with the Nextflow Gradle plugin is to use the `nextflow plugin create` sub-command, which creates a plugin project based on the [Nextflow plugin template](https://github.com/nextflow-io/nf-plugin-template/), which in turn uses the Gradle plugin.
+
+To create a Nextflow plugin with the Gradle plugin, simply run `nextflow plugin create` on the command line. It will prompt you for your plugin name, organization name, and project path.
```

Comment:
```suggestion
To create a Nextflow plugin with the Gradle plugin, run `nextflow plugin create` on the command line. It will prompt you for your plugin name, organization name, and project path.
```